### PR TITLE
Add `is_scaled` to `Templates` object

### PR DESCRIPTION
**Pull request by h-mayorquin**
Shouod solve https://github.com/SpikeInterface/spikeinterface/issues/2831

Draft because I don't know what will fail : )

### CODE REVIEW

File: src/spikeinterface/core/template.py [hunk 1]
```diff
@@ -30,9 +30,11 @@ class Templates:
         Array of unit IDs. If `None`, defaults to an array of increasing integers.
     probe: Probe, default: None
         A `probeinterface.Probe` object
+    is_scaled : bool, optional default: True
+        If True, it means that the templates are in uV, otherwise they are in raw ADC values.
     check_for_consistent_sparsity : bool, optional default: None
         When passing a sparsity_mask, this checks that the templates array is also sparse and that it matches the
-        structure fo the sparsity_masl.
+        structure fo the sparsity_mask. If False, this check is skipped.
```
Comment (zm711) [inline src/spikeinterface/core/template.py:37]:
```suggestion
        structure of the sparsity_mask. If False, this check is skipped.
```

just so we don't forget this tiny typo :)

Comment (h-mayorquin) [reply src/spikeinterface/core/template.py:37]: Thanks Zach!

File: src/spikeinterface/sortingcomponents/clustering/circus.py
Comment (h-mayorquin) [inline src/spikeinterface/sortingcomponents/clustering/circus.py:1]: Thanks for the fix, I was meaning to look for the rest of the examples. I did a short search yesterday but it seems that my regex skills failed me.

**Review by alejoe91 — APPROVED** [review]

**Comment by alejoe91** [comment]
@h-mayorquin this is good to merge for me!

**Pull request merged**